iptables/iptables.go

```diff
@@ -63,6 +63,12 @@ func (e ChainError) Error() string {
 
 func initCheck() error {
 	if iptablesPath == "" {
+		if out, err := exec.Command("modprobe", "-va", "nf_nat").CombinedOutput(); err != nil {
```

Comment:
Actually `initCheck` is being called in `raw` (and `Exists`),  which guarantees the probing is done by any iptables API call.
Thanks, I will wrap it with once, to avoid race.